lib/src/service/entities/status.dart

```diff
@@ -53,7 +49,7 @@ class Status with _$Status {
     Language? language,
 
     /// ID of the status being replied.
-    String? inReplyToTweetId,
+    String? inReplyToId,
```

Comment:
Wow, I could not realized there was such a typo mixed in! This is a breaking change, but it's a very important one :)